evm-contracts/test/v0.6/EACAggregatorProxy.test.ts

```diff
@@ -206,6 +210,19 @@ describe('EACAggregatorProxy', () => {
         }, 'No proposed aggregator present')
       })
     })
+
+    describe('when read from a contract that is not permissioned', () => {
+      beforeEach(async () => {
+        testHelper = await testHelperFactory.connect(personas.Carol).deploy()
```

Comment:
I'm not sure how this works, but it seems an `afterEach()` where you `"disconnect"` from the contract would be help.

Reply:
the `connect` name is a confusing. No persistent connection or similar is created, so there's no need to clean anything up